Mobile Buy SDK Sample Apps/Advanced App - ObjC/Mobile Buy SDK watchOS Sample App Extension/InterfaceController.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  InterfaceController.swift
+//  Mobile Buy SDK watchOS Sample App Extension
+//
+//  Created by Shopify.
+//  Copyright (c) 2016 Shopify Inc. All rights reserved.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+//
+
+import WatchKit
+import Foundation
+
+public struct Constants {
+    static let handoffActivityType = "com.shopify.Mobile-Buy-SDK-Advanced-Sample.view.product"
+}
+
+class InterfaceController: WKInterfaceController {
+
+    /* ---------------------------------
+     ** Configure store credentials to
+     ** use with your specific store.
+     */
+    let shopDomain:     String = ""
+    let apiKey:         String = ""
+    let appID:          String = ""
+    let merchant_id:    String = ""
```

Comment:
these could be pulled from a single source.  Also, inconsistency between `ID` and `_id`